llama-index-integrations/graph_stores/llama-index-graph-stores-kuzu/pyproject.toml

```diff
@@ -27,11 +27,11 @@ exclude = ["**/BUILD"]
 license = "MIT"
 name = "llama-index-graph-stores-kuzu"
 readme = "README.md"
-version = "0.2.0"
+version = "0.6.0"
```

Comment:
lol 2 to 6? Sure, why not :) 

Lets also update the `[tool.llamahub.class_authors]` to add the new class

Reply:
(ah, the version matches the kuzu version)

Reply:
Actually, I'm not sure what the graph store's version should be 😅.. Should I just change it back to 0.3.0? I don't see us bumping the version for every kuzu version.

Reply:
Done! Updated 👍🏽